keyboards/percent/booster/keymaps/default/keymap.c

```diff
@@ -0,0 +1,27 @@
+/* Copyright 2019 REPLACE_WITH_YOUR_NAME
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    [0] = LAYOUT_numpad_5x4(
+        KC_NLCK, KC_SLSH, KC_ASTR, KC_MINS,
```

Comment:
Looks like a kbfirmware bug of sorts here; these aren't numpad keycodes.

```suggestion
        KC_NLCK, KC_PSLS, KC_PAST, KC_PMNS,
```